src/components/testimonials/Testimonials.tsx

```diff
@@ -0,0 +1,62 @@
+import "./testimonials.scss";
+
+type Testimonial = {
+  id: number;
+  name: string;
+  title: string;
+  img: string;
+  desc: string;
+  featured?: boolean;
+};
+
+export default function Testimonials() {
+  const data: Testimonial[] = [
+    {
+      id: 1,
+      name: "Hi-C Costa Rica",
+      title: "Beverage Company",
+      img:
+        "#",
```

Comment:
can we add the links from cloudinary here already?

Reply:
Done!